access_application_test.go

```diff
@@ -46,13 +48,15 @@ func TestAccessApplications(t *testing.T) {
 	updatedAt, _ := time.Parse(time.RFC3339, "2014-01-01T05:20:00.12345Z")
 
 	want := []AccessApplication{AccessApplication{
-		ID:              "480f4f69-1a28-4fdd-9240-1ed29f0ac1db",
-		CreatedAt:       &createdAt,
-		UpdatedAt:       &updatedAt,
-		AUD:             "737646a56ab1df6ec9bddc7e5ca84eaf3b0768850f3ffb5d74f1534911fe3893",
-		Name:            "Admin Site",
-		Domain:          "test.example.com/admin",
-		SessionDuration: "24h",
+		ID:                     "480f4f69-1a28-4fdd-9240-1ed29f0ac1db",
+		CreatedAt:              &createdAt,
+		UpdatedAt:              &updatedAt,
+		AUD:                    "737646a56ab1df6ec9bddc7e5ca84eaf3b0768850f3ffb5d74f1534911fe3893",
+		Name:                   "Admin Site",
+		Domain:                 "test.example.com/admin",
+		SessionDuration:        "24h",
+		AllowedIdps:            []string{"f174e90a-fafe-4643-bbbc-4a0ed4fc8415"},
```

Comment:
I took this IdP from another unit-test file (https://github.com/cloudflare/cloudflare-go/blob/master/access_identity_provider_test.go#L24), I wasn't sure what was configured in the test env and how really test this code. Any pointers / comments are welcome.